rest/taskrouter/activities/instance/delete/example-1/example-1.5.x.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library
+
+use Twilio\Rest\Client;
```

Comment:
Based on the code used in the tutorial code of task-router-laravel this is how i should use the import, and PHPStorm formats it this way. It do works with a file with a namespace, e.g, https://github.com/TwilioDevEd/task-router-laravel/blob/master/app/TaskRouter/WorkspaceFacade.php.